Source/TemplateLib/CommonSqlCode.cs

```diff
@@ -3616,10 +3621,15 @@ public string GetColumnEnumAttributeParams(ColumnSchema column)
         /// <param name="column"></param>
         public string GetColumnEnumAttributeParams(ViewColumnSchema column)
         {
-            return string.Format("\"{0}\", typeof({1}), System.Data.{2}, ",
+            string defaultValue = GetCSDefaultValueByType(column, true);
+
+            return string.Format("\"{0}\", typeof({1}), System.Data.{2}, /*precision*/ {3}, /*scale*/ {4}, /*default*/ {5}, ",

```

Comment:
Changes in this file scare me a little bit. I'm not 100% sure where this method call is being consumed by, but I remember the nightmares to add oracle support and it was very touchy and I know the precisions and scales weren't always right or expected when you are from a SQL Server world.

Reply:
Currently the biggest changes in this file are in support of the new attributes added to the Column Enum.  Those should not be touching any actual functionality, just presenting users with more data about the database if it exists.  The other two changes were submitted by other users years ago and looked common sense, so I included them.  It would be nice if I could setup some sort of environment to generate against several different db types for final testing.